types/file-manager/index.d.ts

```diff
@@ -0,0 +1,82 @@
+// Type definitions for file-manager 2.0
+// Project: https://github.com/DoctorMcKay/node-file-manager#readme
+// Definitions by: repistory <https://github.com/repository>
```

Comment:
```suggestion
// Definitions by: repistory <https://github.com/repository>
//                 Joshua Jeschek <https://github.com/joshuajeschek>
```
If you want to, you can add me as a co-owner, so that future changes are easier.